include/modem/lte_lc.h

```diff
@@ -182,6 +188,104 @@ struct lte_lc_cell {
 	uint32_t tac;	/* Tracking Area Code */
 };
 
+struct lte_lc_current_cell {
+	/** Mobile Country Code. */
+	int mcc;
+
+	/** Mobile Newtork Code. */
+	int mnc;
+
+	/** E-UTRAN cell ID. */
+	uint32_t cell_id;
+
+	/** Tracking are code. */
+	uint32_t tac;
+
+	/** EARFCN of the neighbour cell, per 3GPP TS 36.101. */
+	uint32_t earfcn;
+
+	/** Timing advance decimal value.
+	 *  Range [0..20512, TIMING_ADVANCE_NOT_VALID = 65535]. */
+	uint16_t timing_advance;
+
+	/** Measurement time of serving cell in milliseconds.
+	 * Range 0 - 18 446 744 073 709 551 614 ms.
+	 */
+	uint32_t measurement_time;
+
+	/** Physical cell ID. */
+	uint16_t phys_cell_id;
+
+	/** RSRP of the neighbor celll.
+	 *  -17: RSRP < -156 dBm
+	 *  -16: -156 ≤ RSRP < -155 dBm
+	 *  ...
+	 *  -3: -143 ≤ RSRP < -142 dBm
+	 *  -2: -142 ≤ RSRP < -141 dBm
+	 *  -1: -141 ≤ RSRP < -140 dBm
+	 *  0: RSRP < -140 dBm
+	 *  1: -140 ≤ RSRP < -139 dBm
+	 *  2: -139 ≤ RSRP < -138 dBm
+	 *  ...
+	 *  95: -46 ≤ RSRP < -45 dBm
+	 *  96: -45 ≤ RSRP < -44 dBm
+	 *  97: -44 ≤ RSRP dBm
+	 *  255: not known or not detectable
```

Comment:
I'm not sure about these RSRP and RSRQ members. Ideally, we should convert them, but because they are by definition ranges, it's not straight-forward.
If we keep it like this, it needs to be documented in some other way, as this current format doesn't look good in the generated docs.

Reply:
This is handled the same way in https://github.com/nrfconnect/sdk-nrf/pull/4345. I think the suggested alternatives there was either keep it as is (mapping in API doc) or just refer to the API command specification.